README.md

```diff
@@ -317,6 +318,21 @@ GRUB_CMDLINE_LINUX_DEFAULT="quiet splash initcall_blacklist=amd_pstate_init amd_
 
 After you are done, run `sudo update-grub` or `sudo grub-mkconfig -o /boot/grub/grub.cfg`, if you are using Arch.
 
+### AUR
+
+* The command ```sudo auto-cpufreq --install``` produces error [#471](https://github.com/AdnanHodzic/auto-cpufreq/issues/471) please don't use it.
+    * This script is supposed to automate the process of enabling auto-cpufreq.service so you need to manually open terminal and type
+    ~~~
+    sudo systemctl enable --now auto-cpufreq.service
+    ~~~
+    for the service to work.
+* If you have Power Profiles Daemon installed you are likely to experience auto-cpufreq.serivce not autostarting after each reboot like the issue [#463](https://github.com/AdnanHodzic/auto-cpufreq/issues/463). This happens due to it's conflict with power-profile-daemon.service. 
```

Comment:
> auto-cpufreq.serivce
Typo

Reply:
If you have Power Profiles Daemon installed you are likely to experience auto-cpufreq.serivce not autostarting after each reboot like the issue [#463](https://github.com/AdnanHodzic/auto-cpufreq/issues/463). This happens due to it's conflict with power-profile-daemon.service.

This is already [mentioned here](https://github.com/AdnanHodzic/auto-cpufreq#1-power_helperpy-script-snap-package-install-only). So let's make a link to this line, and rephrase the text something along the lines: "Power Profiles Daemon is [automatically disabled by auto-cpufreq-installer](https://github.com/AdnanHodzic/auto-cpufreq#1-power_helperpy-script-snap-package-install-only). However this doesn't happen with AUR package and will lead to problems (i.e: [#463]((https://github.com/AdnanHodzic/auto-cpufreq/issues/463))) if not disabled manually.

Reply:
Please review the changes I have fixed the typo and rephrased the line as per suggestion.